packages/flame_audio/lib/bgm.dart

```diff
@@ -3,19 +3,28 @@ import 'dart:io';
 import 'package:audioplayers/audioplayers.dart';
 import 'package:flutter/widgets.dart';
 
+/// {@template _bgm}
```

Comment:
just curious, what is this?

Reply:
It allows for docs to be reused, if you look below, it is used on the constructor with `@macro _bgm`